features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/mysql.sql

```diff
@@ -1130,9 +1129,7 @@ CREATE TABLE IF NOT EXISTS IDN_CORS_ORIGIN (
     TENANT_ID         INT           NOT NULL,
     ORIGIN            VARCHAR(2048) NOT NULL,
     UUID              CHAR(36)      NOT NULL,
-
     PRIMARY KEY (ID),
-    UNIQUE (TENANT_ID, ORIGIN),
```

Comment:
Shall we add this `UNIQUE (TENANT_ID, ORIGIN),` removal change to mssql script as well?